docs/RFCS/20211106_multitenant_cluster_settings.md

```diff
@@ -59,42 +61,78 @@ We propose splitting the cluster settings into three *classes*:
 
 1. System only (`system`)
 
-   Settings associated with the host cluster, only usable by the system tenant.
+   Settings associated with the storage cluster, only usable by the system tenant.
    These settings are not visible at all from other tenants. Settings code
    prevents use of values for these settings from a tenant process.
 
    Example: `kv.allocator.qps_rebalance_threshold`.
-  
+
 2. Tenant read-only (`tenant-ro`)
 
    These settings are visible from non-system tenants but the tenants cannot
    modify the values.
 
-   New SQL syntax allows the system tenant to set the value for a specific
-   tenant; this results in the tenant (asynchronously) getting the updated
-   value. The system tenant can also set an "all tenants" default value with a
-   single command; the value applies to all tenants that don't have their own
-   specific value, including future tenants.
+   The observed value of settings in this class is:
+
+   - by default, the value held for the setting in the system tenant
+     (the storage cluster's value in the system tenant's
+     `system.settings`).
+   - New SQL syntax allows the system tenant to set the value for a specific
+     tenant; this results in the tenant (asynchronously) getting the updated
+     value. (i.e. the value for one tenant can be overridden away from the default)
+
+   Examples:
 
-   Examples: `kv.bulk_ingest.batch_size`, `tenant_cpu_usage_allowance`.
+   - Settings that affect the KV layer, should not be writable by tenants,
+     but which benefit the SQL layer in tenants: `kv.raft.command.max_size`.
 
-3. Tenant writable (`tenant-rw`)
+   - Settings that benefit from being overridden per tenant, but where
```

Comment:
To me these aren't so much two distinct categories, but rather one category, plus the same override-capability we have for all tenant-* settings, ro or rw.

I'd describe this category as:
- Settings where exposing the system tenant's current value -- whether that is coming from its default or has been configured explicitly -- is desirable. For example, clients typically try to limit the size of batches they send to stay within the limits of what the host cluster is configured to process, making it useful to know the current values of  `kv.raft.command.max_size` and `kv.bulk_ingest.batch_size` in the host cluster. 

- As with all tenant-exposed setting, the value read by a specific or all tenants can be forcibly overridden from the host, which could be useful, for example, as a mitigation measure in an incident to force a tenant who is running several large imports to use a smaller batch size to stop OOMs.

As I sit here and think about it: Is there really any reason `tenant_cpu_usage_allowance` should actually be tenant-ro versus tenant-rw? Seems like it doesn't really matter that it be tenant-ro if we're just going to override it anyway.